electricitylci/coal_upstream.py

```diff
@@ -83,17 +83,17 @@ def read_eia923_fuel_receipts(year):
         csv_file = [f for f in all_files
                     if '.csv' in f
                     and '_page_5_reduced.csv' in f]
-        csv_path = os.path.join(expected_923_folder,csv_file[0])
-        if os.path.exists(csv_path):
-            eia_fuel_receipts_df=pd.read_csv(csv_path)
+        if csv_file != []:
```

Comment:
Can change this to just 
```python
if csv_file:
```
An empty list will return false, a list with objects will return true.